### DISCUSSION TITLE
Option to regenerate thumbnails only for videos

### DISCUSSION THREAD

**Discussion by ilopmar** [immich-app/immich#10294] [Feature Request]
### The bug

After the new feature included in v1.106.x I've tried manually regenerating the thumbnails of some videos. No more "black" thumbnail anymore, thanks! :clap: 
The problem is that to apply to all my videos I would need to force the generation of the thumbnails for all my assets, and there are quite a few: Almost 350GB that include 1000 videos en +80000 pictures (according to the Server stats).
I don't really want to regenerate ALL the thumbnails for all the pictures for just only 1000 videos.

So, this is a feature request to provide a way to regenerate the thumbnails per type of assets: pictures and videos.

### The OS that Immich Server is running on

Ubunbu Server 22.04.4

### Version of Immich Server

v1.106.4

### Version of Immich Mobile App

v1.106

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Not necessary for this
```

### Your .env content

```Shell
Not necessary for this
```

### Reproduction steps

```bash
Already explained
```

**Comment by mertalev** [comment]
This isn't to dismiss the feature request itself, but a solution for your case could be to use search to filter for videos, select the ones that have black thumbnails and regenerate those specifically.

**Reply by ilopmar** [reply]
Thanks, that did the trick! Another option I was considering was getting the asset ids directly from the database and write a bash script to send a request to the API to regenerate the thumbnails. At the end, as there wasn't many with black thumbnails I did what you suggested
